x-pack/plugins/reporting/common/errors/errors.test.ts

```diff
@@ -1,33 +0,0 @@
-/*
```

Comment:
@rshen91 this file accidentally got deleted instead of being moved to `packages/kbn-reporting/common`